packages/-ember-data/config/ember-try.js

```diff
@@ -14,6 +14,13 @@ module.exports = function() {
           },
           npm: {},
         },
+        {
+          name: 'with-safari',
+          env: {
+            TARGET_SAFARI: true,
```

Comment:
we shouldn't need this since we've got the other test runs specified with this arg in azure and GitHub CI